WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowIntroViewController.swift

```diff
@@ -145,8 +147,14 @@ class BloggingRemindersFlowIntroViewController: UIViewController {
     @objc private func getStartedTapped() {
         tracker.buttonPressed(button: .continue, screen: .main)
 
-        let flowSettingsViewController = BloggingRemindersFlowSettingsViewController(tracker: tracker)
-        navigationController?.pushViewController(flowSettingsViewController, animated: true)
+        do {
+            let flowSettingsViewController = try BloggingRemindersFlowSettingsViewController(blogURIRepresentation: blogURIRepresentation, tracker: tracker)
```

Comment:
I feel like it's unusual to have instantiating a view controller throw an error. I think it might be best to continue to show the view controller, but it can display an error if it's been unable to create the scheduler.

Reply:
As chatted via DM... the reason this was done this way is because it's an error scenario that's very unlikely to happen, but that would prevent the flow from working fine.  So my thinking is I didn't want to let the user go forward as if nothing happened.

By interrupting the flow I'm hoping we'll get at least reports of an issue.